Controller/home.py

```diff
@@ -4,7 +4,6 @@
 from werkzeug.utils import redirect
 from Controller.user_controller import User
 from Controller.application_controller import Application
-from Controller.email_framework import *
```

Comment:
Please only import necessary functions and classes, as it would pollute the name space

Reply:
This was redundant. Hence was removed.